pkg/peertls/tlsopts/tls_test.go

```diff
@@ -58,3 +66,97 @@ func TestVerifyIdentity_error(t *testing.T) {
 		})
 	}
 }
+
+func TestExtensionMap_HandleExtensions(t *testing.T) {
+	// TODO: rename and/or move test
```

Comment:
maybe this and the test below should be moved to revocation_db_test.go since it seems like they're testing if the revocations are working properly?